librarian/data/storage.py

```diff
@@ -160,22 +161,25 @@ def name(self):
         return self.dev.name
 
     @property
-    def total_content_size(self):
-        success, size = exts.fsal.get_path_size(self.path)
-        if not success:
-            raise StorageError(self.id, 'Could not get content size')
-        return size
+    def total(self):
+        return self.stat.f_bsize * self.stat.f_blocks
 
     @property
-    def stat(self):
-        return self.dev.stat
+    def free(self):
+        return self.stat.f_bsize * self.stat.f_bfree
```

Comment:
I believe this should be `f_bavail`. I forgot the details but there is some difference.